src/NzbDrone.Core/Download/Aggregation/Aggregators/AggregateLanguages.cs

```diff
@@ -66,6 +70,23 @@ public RemoteMovie Aggregate(RemoteMovie remoteMovie)
                 languages = languages.Except(languagesToRemove).ToList();
             }
 
+            if (languages.Count == 1 && languages.First() == Language.German)
```

Comment:
Any reason we are doing this in the Aggregation class instead of in the Language parser? Seems like it should be there instead of here

Reply:
You are right, I first thought that it has to go there because at that point we have the Original language metadata, but I can just use `Language.Original` and it gets replaced later on. Updated it.